src/mlpack/tests/ann/layer/ftswish.cpp

```diff
@@ -46,16 +46,16 @@ TEST_CASE("FTSwishTest", "[ANNLayerTest]")
   // Test the Forward function
   REQUIRE(abs(arma::accu(output - actualOutput)) <= 0.0001);
 
-  arma::mat delta = {{0.0 ,0.84327731, 0.0},
-    {0.91985943, 1.05058055 ,0.0 },
-    {0.0, 1.08399125 ,1.00053333}};
+  arma::mat delta = {{0.615941, 0.989097, 0.0},
+    {1.03315, 1.09083 ,0.0 },
+    {0.0, 1.07286 ,1.00045}};
```

Comment:
Just out of curiosity, how did you get the new values here?

Reply:
hand-calculated in python.  used the function:
```python
def d_ftswish(x):
    return 1.0 / (1 + exp(-x)) * (1 + x * (1.0-1.0 / (1 + exp(-x)))) if x > 0 else 0
```